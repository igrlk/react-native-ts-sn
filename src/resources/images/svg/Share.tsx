import React from 'react';
import Svg, { Path } from 'react-native-svg';

export default function Share() {
	return (
		<Svg width='22' height='22' viewBox='0 0 22 22' fill='none'>
			<Path
				d='M17.4319 14.1337C16.1665 14.1337 15.0382 14.7367 14.3164 15.6686L8.27281 12.2928C8.41442 11.8862 8.49208 11.4477 8.49208 10.9954C8.49208 10.5386 8.41442 10.1047 8.26825 9.69352L14.3073 6.32226C15.0245 7.25872 16.1574 7.86628 17.4273 7.86628C19.5926 7.86628 21.3604 6.10299 21.3604 3.93314C21.3604 1.76329 19.5972 0 17.4273 0C15.2574 0 13.4942 1.76329 13.4942 3.93314C13.4942 4.38995 13.5718 4.82849 13.718 5.23505L7.68353 8.60631C6.96633 7.66528 5.83344 7.06229 4.56351 7.06229C2.39823 7.06229 0.630371 8.82558 0.630371 10.9954C0.630371 13.1653 2.39823 14.9286 4.56808 14.9286C5.83801 14.9286 6.9709 14.321 7.69266 13.38L13.7317 16.7558C13.5855 17.1669 13.5033 17.6101 13.5033 18.0669C13.5033 20.2321 15.2666 22 17.4364 22C19.6063 22 21.3696 20.2367 21.3696 18.0669C21.3696 15.897 19.6017 14.1337 17.4319 14.1337ZM17.4319 1.23796C18.9211 1.23796 20.1316 2.4485 20.1316 3.93771C20.1316 5.42691 18.9211 6.63746 17.4319 6.63746C15.9427 6.63746 14.7321 5.42691 14.7321 3.93771C14.7321 2.4485 15.9472 1.23796 17.4319 1.23796ZM4.56808 13.6952C3.07888 13.6952 1.86833 12.4846 1.86833 10.9954C1.86833 9.50623 3.07888 8.29568 4.56808 8.29568C6.05728 8.29568 7.26783 9.50623 7.26783 10.9954C7.26783 12.4846 6.05271 13.6952 4.56808 13.6952ZM17.4319 20.762C15.9427 20.762 14.7321 19.5515 14.7321 18.0623C14.7321 16.5731 15.9427 15.3625 17.4319 15.3625C18.9211 15.3625 20.1316 16.5731 20.1316 18.0623C20.1316 19.5515 18.9211 20.762 17.4319 20.762Z'
				fill='#AAAAAA'
			/>
		</Svg>
	);
}
