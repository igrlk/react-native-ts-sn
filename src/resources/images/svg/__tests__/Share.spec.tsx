import React from 'react';
import renderer from 'react-test-renderer';

import Share from '../Share';

describe('Share', () => {
	it('match to snapshot', () => {
		const tree = renderer.create(<Share />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
