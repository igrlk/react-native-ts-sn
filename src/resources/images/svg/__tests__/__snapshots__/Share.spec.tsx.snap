// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Share match to snapshot 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="22"
  bbWidth="22"
  fill="none"
  focusable={false}
  height="22"
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      undefined,
      Object {
        "opacity": 1,
      },
      Object {
        "flex": 0,
        "height": 22,
        "width": 22,
      },
    ]
  }
  vbHeight={22}
  vbWidth={22}
  width="22"
>
  <RNSVGGroup
    fill={null}
    fillOpacity={1}
    fillRule={1}
    font={Object {}}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={
      Array [
        "fill",
      ]
    }
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  >
    <RNSVGPath
      d="M17.4319 14.1337C16.1665 14.1337 15.0382 14.7367 14.3164 15.6686L8.27281 12.2928C8.41442 11.8862 8.49208 11.4477 8.49208 10.9954C8.49208 10.5386 8.41442 10.1047 8.26825 9.69352L14.3073 6.32226C15.0245 7.25872 16.1574 7.86628 17.4273 7.86628C19.5926 7.86628 21.3604 6.10299 21.3604 3.93314C21.3604 1.76329 19.5972 0 17.4273 0C15.2574 0 13.4942 1.76329 13.4942 3.93314C13.4942 4.38995 13.5718 4.82849 13.718 5.23505L7.68353 8.60631C6.96633 7.66528 5.83344 7.06229 4.56351 7.06229C2.39823 7.06229 0.630371 8.82558 0.630371 10.9954C0.630371 13.1653 2.39823 14.9286 4.56808 14.9286C5.83801 14.9286 6.9709 14.321 7.69266 13.38L13.7317 16.7558C13.5855 17.1669 13.5033 17.6101 13.5033 18.0669C13.5033 20.2321 15.2666 22 17.4364 22C19.6063 22 21.3696 20.2367 21.3696 18.0669C21.3696 15.897 19.6017 14.1337 17.4319 14.1337ZM17.4319 1.23796C18.9211 1.23796 20.1316 2.4485 20.1316 3.93771C20.1316 5.42691 18.9211 6.63746 17.4319 6.63746C15.9427 6.63746 14.7321 5.42691 14.7321 3.93771C14.7321 2.4485 15.9472 1.23796 17.4319 1.23796ZM4.56808 13.6952C3.07888 13.6952 1.86833 12.4846 1.86833 10.9954C1.86833 9.50623 3.07888 8.29568 4.56808 8.29568C6.05728 8.29568 7.26783 9.50623 7.26783 10.9954C7.26783 12.4846 6.05271 13.6952 4.56808 13.6952ZM17.4319 20.762C15.9427 20.762 14.7321 19.5515 14.7321 18.0623C14.7321 16.5731 15.9427 15.3625 17.4319 15.3625C18.9211 15.3625 20.1316 16.5731 20.1316 18.0623C20.1316 19.5515 18.9211 20.762 17.4319 20.762Z"
      fill={
        Array [
          0,
          4289374890,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      x={0}
      y={0}
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
